industrydive/django-ckeditor
# 2050 ckeditor pullquote plugin (closes #14 in django-ckeditor repo)

#### Which issue(s) does this PR address?
closes #14  (named 2050 to match up with the divesite issue number: industrydive/divesite#2050)

#### What does this PR do?
Pulls in a new directory in the ckeditor plugin dir, which makes a pull quote plugin for editors to use. It also makes minor edits to the diveimage plugin so they play nice.

#### Where should the reviewer start?
**Make sure the equivalent divesite PR (https://github.com/industrydive/divesite/pull/2162) has been resolved if it can be, or this PR you are reviewing wont work (needs some changes to the settings.py file that the divesite pr provides)**

Go make a new newspost, and scroll down to the editor and look for the icon that looks like 
a speech bubble (quote .etc), and click it. 

This should open up a dialog in which you can provide a quote, image url, speaker name, and speaker title to create a pullquote.

#### How should this be manually tested?
Create that SOB and then view the newspost and see how she looks. It should look like a well-styled pullquote (see screenshots below)

- Test it in desktop,
- Test it in mobile,
- Open the editor back up and make another pullquote
- add some text to this editor
- rearrange the pull quotes in the editors
- edit the text and image in the pullquote
- leave an image field blank and close the dialog and see if it works (no image in the pullquote in the editor)
- save the page 
- view the actual newspost again and make sure everything looks peachy

#### Screenshots
- good desktop pullquote

![](https://cloud.githubusercontent.com/assets/10449847/17599794/c4b51118-5fce-11e6-9b74-5347f9cf1968.png)
